test/tests.js

```diff
@@ -1554,7 +1554,7 @@ USD,
         mParticle.forwarder.process({
             EventName: 'eCommerce - AddToCart',
             EventDataType: MessageType.Commerce,
-            EventCategory: CommerceEventType.AddToCart,
+            EventCategory: CommerceEventType.ProductAddToCart,
```

Comment:
Can you also reference `mParticle.CommerceEventType` here as well?

Reply:
Yes.  I think the reason it is this way is because before this feature and test infrastructure change, MP was stubbed and very simple, and so mParticle didn't have all of its members, including CommerceEventType.